aws/naming_resources.md

```diff
@@ -0,0 +1,10 @@
+# Naming Resources
+
+## IAM Resources
+
+AWS operator needs a set of roles in customer accounts to be able to operator against the AWS API. Most of the customers need bounded policy to be security complaint. Here we define a convention to help them to achieve that and a same time have a coherent way to name the roles. From now own all clusters roles we need to create should have the prefix `gs-`. Example:
+
+OLD worker role: `5md7e-worker-EC2-K8S-Role`
+NEW worker role: `gs-5md7e-worker-EC2-K8S-Role`
```

Comment:
Can we write it down schematically next to having examples? 

```
gs-[cluster id]-[what is this?]-EC2-K8S-Role
```

Reply:
I would really like to see that scheme in there. Three people liked it apparently so let's get it in as is.

Reply:
But this is the schema, right?
https://github.com/giantswarm/fmt/pull/105/files#diff-1d117290cfad3e20761fe775d42fff1aR9-R11

Reply:
ok I made it more explicit, does it better?